app/src/main/res/layout/activity_offer.xml

```diff
@@ -13,7 +13,7 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:clipToPadding="false"
-        android:paddingBottom="120dp"
+        android:layout_above="@+id/signButton"
```

Comment:
Whoa, does this work? 🤯 I thought `layout_above` was a `RelativeLayout`-only attribute? 🤔

Reply:
Thought so too, you learn something new everyday :D